src/csharp/PesterTests/ContainerInfoTests.cs

```diff
@@ -0,0 +1,21 @@
+namespace PesterTests
+{
+    using Pester;
```

Comment:
Usings in namespace, is a style choice I don't like.

Reply:
I got some warning around this, prefer the other convention too.  I'll revert as it doesn't matter in terms of functionality.  Using Roslynator to scan.